### DISCUSSION TITLE
Organizationの月間ランキングの仕様について

### DISCUSSION THREAD

**Discussion by KenUehara0376255** [increments/qiita-discussions#624] [ヘルプ]
三菱ケミカル株式会社の植原と申します。

弊社でQiitaの[Organization](https://qiita.com/organizations/mitsubishichemical)を運営しているのですが、月間ランキングに表示されている件数と実際に投稿されている件数に差異があるように見えます。

2月中に投稿した件数は6件なのですが、月間ランキングを参照すると3件と表示されています。
<img width="957" alt="スクリーンショット 2024-02-29 19 06 04" src="https://github.com/increments/qiita-discussions/assets/134242003/47055574-dd04-46ac-bb50-37f217b3af46">

同様に週間ランキングも参照してみましたが、こちらも実績値は4件のはずなのですが、週間ランキング上では3件と表示されています。

実績値として各月の投稿件数、月間ランキングを計測していますので、月間ランキングがどのような仕様なのかをご教授いただけますと幸いです。

**Comment by tanaka-asami415** [comment]
@KenUehara0376255 
お問合せいただきありがとうございます。

https://qiita.com/organizations ページのランキングは、記事の `作成日`（月間は30日以内に作成された記事）で集計されているのですが
お調べしたところ、投稿された6記事中 3記事は限定共有記事として投稿されており、`作成日`が30日より前（公開は30日以内）となっておりました。
そのため、3件の表示となっています。

`公開日` で集計する方が適切だと思われるため、今後修正予定です🙇

**Reply by KenUehara0376255** [reply]
@tanaka-asami415 
調査いただきありがとうございます。
そういった仕様であれば承知しました。今後ともよろしくお願いいたします。

**Comment by kimkim0814** [comment]
@KenUehara0376255
お待たせいたしました!

Organizationの月間ランキングの仕様の方を修正いたしました!
`作成日`基準で集計されていましたが、本日より`公開日`基準で集計されるようになりました。

この度はご意見をいただきありがとうございました。
